web/config/app.go

```diff
@@ -0,0 +1,98 @@
+package config
```

Comment:
Feels like this file belongs in the `scm` package as `github_app.go`

Reply:
Would also be nice to have a `TestGitHubApp` thing in `github_app_test.go`

Reply:
Moved, added test